hw2/hw2_1/main.js

```diff
@@ -0,0 +1,129 @@
+const express = require('express');
+const fs = require('fs');
+
+const app = express();
+const jsonParser = express.json();
+
+const port = 8080;
+
+app.use(express.static('public'));
+
+const filePath = 'hw2/hw2_1/users.json';
+
+app.get('/api/users', (req, res) => {
+  if (!req.query.name || !req.query.limit) return res.status(404).end('Oops... We need parameters NAME and LIMIT');
+
+  const contentJSON = fs.readFileSync(filePath, 'utf8');
```

Comment:
It't better to read a file asynchronously in this case since reading a file sync can slow down an application.

Reply:
Here you can just create users just in a memory.